public/home.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://code.jquery.com/jquery-3.6.0.min.js"
+      integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4="
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <div style="position: relative" id="header">
+      <div class="custom-shape-divider-top-1627313518">
+        <svg
+          data-name="Layer 1"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 1200 120"
+          preserveAspectRatio="none"
+        >
+          <path
+            d="M985.66,92.83C906.67,72,823.78,31,743.84,14.19c-82.26-17.34-168.06-16.33-250.45.39-57.84,11.73-114,31.07-172,41.86A600.21,600.21,0,0,1,0,27.35V120H1200V95.8C1132.19,118.92,1055.71,111.31,985.66,92.83Z"
+            class="shape-fill"
+          ></path>
+        </svg>
+      </div>
+
+  
+    </div>
+    <div style="position: relative;">
+        <div class="card" style="background-color: #BCD2E8; border-color:white;color:black; border-radius: 20px; border-width: 5px; width: 90%; text-align: center;top:20px; left: 90px; ">
+            <div class="card-body">
+              <h1>Survey Home Page</h1>
+            </div>
+          </div>
+      
+    </div>
+
+    <div style="position: relative;">
+        <div class="card" style="background-color: #BCD2E8; border-color:white;color: black; border-radius: 20px; border-width: 5px; width: 50%; text-align: center;top:40px; left: 90px; ">
+            <div class="card-body">
+                <a type="button" id="button" class="btn btn-primary btn-large" style="background-color: #BCD2E8; border-color:#BCD2E8;color: black;" href="">
+                    <h3>Click Here To Go To Survey</h3>
+                   </a> 
+            </div>
+
+            
+          </div>
+      
+    </div>
+
+    <div style="position: relative;">
+        <div class="card" style="background-color: #BCD2E8; border-color:white;color: black; border-radius: 20px; border-width: 5px; width: 30%; text-align: center;top:85px; left: 90px; ">
+            <div class="card-body">
+                
+                  <h2>Created By:Aiden,Kabir,Savith</h2>
+            </div>
+          </div>
+      
+    </div>
+
+    <div style="position: relative;">
+        <div class="card" style="background-color: #BCD2E8; border-color:white;color: black; border-radius: 20px; border-width: 5px; width: 36%; text-align: center;top:-185px; left: 850px; ">
```

Comment:
You should also create a class for the styling of all these cards, instead of using inline styling